src/Tools/Model/PropertyManager.php

```diff
@@ -32,6 +32,17 @@ public function getProperty(string $name, $default = null)
         }
     }
 
+    public function unsetProperty(string $name)
```

Comment:
We may could think about the naming scheme for functions like this. removeProperty sounds like more work than just "unset" and may hint that it's connected with an actual database query.